elcid/views.py

```diff
@@ -100,3 +104,105 @@ def get_context_data(self, *args, **kwargs):
             pass
 
         return ctd
+
+
+def ward_sort_key(ward):
+    """"
+    Example wards are
+            "8 West",
+            "PITU",
+            "ICU 4 East",
+            "9 East",
+            "8 South",
+            "9 North",
+            "ICU 4 West",
+            "12 West",
+
+    We want to order them first by number
+    then with the wards that are strings.
+    e.g. ICU
+    """
+
+    start = ward[:2].strip()
+    rest = ward[2:]
+    return (len(start), start, rest)
+
+
+class RenalHandover(LoginRequiredMixin, TemplateView):
+    template_name = "elcid/renal_handover.html"
+
+    def get_unit_ward(self, location):
```

Comment:
The terminology is taken from their word document template? 
I think it's odd terminology to replicate in code - get location perhaps... 

Are you sure you only want to display this if they have both bed and ward?

Reply:
Great call

Reply:
Changed so it will just show the ward if we are missing a bed